util/CMakeLists.txt

```diff
@@ -22,8 +22,10 @@ if(USE_OPENSSL)
     if(NOT BUILD_THIRDPARTY)
         find_package(OpenSSL REQUIRED)
     endif()
+    find_library(CRYPTO_STATIC_LIBRARY NAMES libcrypto.a HINTS ${OPENSSL_ROOT_DIR}/lib/  REQUIRED)
```

Comment:
same

Reply:
@toly-kournik , thanks for sharing "OPENSSL_USE_STATIC_LIBS", this makes us to link static even it UTT.